modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -801,17 +823,17 @@ bool Visualizer::DrawTrajectories(
                  static_cast<float>(object->drops[0](1)), 0, 1;
   start_point = (*motion_buffer)[0].motion * start_point;
   cv::circle(world_image_, world_point_to_bigimg(start_point), 3,
-    cv::Scalar(127, 127, 127));
+    apollo::perception::gray_color);
 
   for (size_t i = 1; i < count; i++) {
     Eigen::Vector4f end_point;
     end_point << static_cast<float>(object->drops[i](0)),
                  static_cast<float>(object->drops[i](1)), 0, 1;
     cv::circle(world_image_, world_point_to_bigimg(end_point), 3,
-      cv::Scalar(127, 127, 127));
+      apollo::perception::gray_color);
```

Comment:
gray_color is defined as `cv::Scalar gray_color = cv::Scalar(128, 128, 128);` in your previous definition.

Reply:
Yes. 127 was incorrect. (128, 128, 128) is actual gray but human cannot distinguish it.